admob/src_java/com/google/firebase/admob/internal/cpp/BannerViewHelper.java

```diff
@@ -143,42 +139,39 @@ public void initialize(Activity activity) {
   public void destroy(final long callbackDataPtr) {
     // If the Activity isn't initialized, there is nothing to destroy.
     if (mActivity == null) {
+      completeBannerViewFutureCallback(callbackDataPtr,
+        ConstantsHelper.CALLBACK_ERROR_NONE,
+        ConstantsHelper.CALLBACK_ERROR_MESSAGE_NONE);
       return;
```

Comment:
This fixes a bug when calling destroy on an uninitialized BannerView.  Previously it would hang as the future was never completed.